projects/everrise/index.js

```diff
@@ -1,296 +1,142 @@
 const sdk = require('@defillama/sdk');
-const abi = require('./abi.json');
+const http = require('../helper/http');
 const BigNumber = require("bignumber.js");
 const { unwrapUniswapLPs } = require("../helper/unwrapLPs");
+const { getChainTransform } = require("../helper/portedTokens");
 
-const STATS ='0xC4DD716a29357317BdC66d9D9cF2ec02fD995742';
+const zeroAddress = '0x0000000000000000000000000000000000000000'
 const BRIDGE_CONTROLLER = '0x0Dd4A86077dC53D5e4eAE6332CB3C5576Da51281';
-const TOKEN ='0x0cD022ddE27169b20895e0e2B2B8A33B25e63579';
-const TOKEN_AVAX ='0xC3A8d300333BFfE3ddF6166F2Bc84E6d38351BED';
-const STAKING ='0x1490EaA0De0b2D4F9fE0E354A7D99d6C6532be84';
-
-const ethPool2LPs = [
-  {
-    owner: "0xA366820f4781049E0f183697252bc6cbc4fcD9e1",
-    pool: "0x9295c3289a0d924c0437436fa9fca5e34cb8acb3",
-  }, // RISE-ETH
-];
-const bscPool2LPs = [
-  {
-    owner: "0xA366820f4781049E0f183697252bc6cbc4fcD9e1",
-    pool: "0xe880A0be3513B92FFe3563D8ee9F6f7b954E4415",
-  }, // RISE-ETH
-];
-const polygonPool2LPs = [
-  {
-    owner: "0xA366820f4781049E0f183697252bc6cbc4fcD9e1",
-    pool: "0xb24A550E8E764CAE3fCe91c333B5B17CccfFf438",
-  }, // RISE-ETH
-];
-const avaxPool2LPs = [
-  {
-    owner: "0xA366820f4781049E0f183697252bc6cbc4fcD9e1",
-    pool: "0xB7395899318b2DF8EedeF9bBfe840bA99932aA13",
-  }, // RISE-ETH
-];
-const fantomPool2LPs = [
-  {
-    owner: "0xA366820f4781049E0f183697252bc6cbc4fcD9e1",
-    pool: "0x70d1894747aed5090539baDA0E9eD1A67D7e31d7",
-  }, // RISE-ETH
-];
-
-function bscTvl(timestamp, block) {
-  return tvl(timestamp, block, 'bsc');
-}
-
-function polygonTvl(timestamp, block) {
-  return tvl(timestamp, block, 'polygon');
-}
-
-function avalancheTvl(timestamp, block) {
-  return tvl(timestamp, block, 'avax');
-}
-
-function fantomTvl(timestamp, block) {
-  return tvl(timestamp, block, 'fantom');
-}
-
-function polygonStaking(timestamp, block) {
-  return staking(timestamp, block, 'polygon', TOKEN);
-}
-
-function bscStaking(timestamp, block) {
-  return staking(timestamp, block, 'bsc', TOKEN);
+const TOKEN = '0xC17c30e98541188614dF99239cABD40280810cA3';
+const STAKE_HOLDING_API = 'https://app.everrise.com/bridge/api/v1/stats'
+const EVERRISE_TOKEN = '0x0cd022dde27169b20895e0e2b2b8a33b25e63579'
```

Comment:
Oh, this is the old token; should just be `TOKEN` 0xC17c30e98541188614dF99239cABD40280810cA3 above

Isn't picking up the value for that token as I think it pulls it from coingecko/cmc so just waiting for update on their side

Reply:
okay, I changed it from ethereum address to coingecko id to get around this problem

Reply:
Oh cool, didn't know you could do that